influxdb/src/main/scala/akka/stream/alpakka/influxdb/InfluxDBSettings.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.influxdb
+
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.duration.TimeUnit
+
+object InfluxDBSettings {
+  val Default = new InfluxDBSettings(batchSize = 10, TimeUnit.MILLISECONDS)
+
+  def apply(): InfluxDBSettings = Default
+
+}
+
+final case class InfluxDBSettings private (
```

Comment:
To enable binary-compatible evolution, we can't use case classes for settings. Even though the constructor is private, scalac creates `apply` with the fields and `unapply`. We've been using [Kaze Class](https://github.com/ktoso/kaze-class) a lot to convert case classes.